docs/data/tree-view/accessibility/accessibility.md

```diff
@@ -0,0 +1,80 @@
+| <kbd class="key">Arrow Right</kbd> or <kbd class="key">Arrow Left</kbd> when RTL is enabled | <ul><li>If focus is on a collapsed item, expands the item without moving focus</li><li>If focus is on an expanded item, moves focus to the first child</li><li>If focus is on an end item, nothing happens</li></ul> |
+| <kbd class="key">Arrow Left</kbd> or <kbd class="key">Arrow Right</kbd> when RTL is enabled | <ul><li>If focus is on an expanded item, collapses the item</li><li>If focus is on a collapsed item that has a parent, moves focus to its parent</li><li>If focus is on a root item, nothing happens</li></ul>       |
+|                                                                 <kbd class="key">Home</kbd> | Focuses the first item in the tree                                                                                                                                                                                   |
+|                                                                  <kbd class="key">End</kbd> | Focuses the last item in the tree                                                                                                                                                                                    |
+|                                                                   <kbd class="key">\*</kbd> | Toggle row children expansion when grouping cell is focused                                                                                                                                                          |
+
+Type-ahead is supported for single characters. When typing a character, focus moves to the next item with a label that starts with the typed character.
+
+## Selection
+
+The tree view support both single and multi selection. To learn more about the selection API, visit the dedicated page for the [Simple Tree View](/x/react-tree-view/simple-tree-view/selection/) or the [Rich Tree View](/x/react-tree-view/rich-tree-view/selection/).
+
+To read more about the distinction between selection and focus, you can refer to the [WAI-ARIA Authoring Practices guide](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#kbd_focus_vs_selection).
+
+### On single-select trees
+
+When a single-select tree receives focus:
+
+- If none of the items are selected when the tree receives focus, focus is set on the first item.
+- If an item is selected before the tree receives focus, focus is set on the selected item.
+
+### On multi-select trees
+
+When a multi-select tree receives focus:
+
+- If none of the items are selected when the tree receives focus, focus is set on the first item.
+- If an item is selected before the tree receives focus, focus is set on the first of the selected items.
```

Comment:
I double checked, and we are both right, sort of. 
If you select a bunch of items, focus on something else, and then focus back on the TreeView, the item that will be focused is indeed the one you selected last. 
However, if you have a selection state initialized on the first render, and focus on the TreeView, the item focused will be the first one 🙈 

https://github.com/mui/mui-x/assets/72460825/c2e2c2a9-7768-4ca3-a94f-8f63a08d92f7


So maybe something like this makes more sense: 
```suggestion
- If one or more items are selected before the tree receives focus, focus is set on
  - the first selected item if it is the first render
  - the item that was last selected 
```

Reply:
Nice confirmation. 👍 
Or maybe like this?

```suggestion
- If one or more items are selected before the tree receives focus, then focus is set on:
  - the first selected item if it is the first render
  - the item that was last selected otherwise
```